### DISCUSSION TITLE
How to specify the projects supported Python versions.

### DISCUSSION THREAD

**Discussion by shadycuz** [python-poetry/poetry#8785] [Q&A] [answered]
I would like to set the version of Python that is supported by my project. The versions are `3.9` to `3.12.*`. I had originaly tried a wild card but those arent supported correctly? https://github.com/python-poetry/poetry/issues/8418

I tried the following but it did not work:
```
python = ">=3.9,<=3.12"
python = ">=3.9,<=3.12.x"
```

**Comment by edgarrmondragon** [comment, marked as answer]
You probably want `python = ">=3.9,<3.13"`

**Reply by shadycuz** [reply]
I thought I was being clever and went with: `python = ">=3.9,<3.12.100"` 🤣
